common/views/Governor.jsx

```diff
@@ -59,24 +108,27 @@ var Governor = React.createClass({
 
     return (
       <div style={styles.root}>
-        <section>
-          <img src={governor.avatar} />
-          <div className="ui three column grid">
-            <div className="column">
-              <div>{governorStats.notyet || 0}</div>
-              <div>還沒做</div>
+        <section style={styles.section}>
+          <Avatar style={styles.avatar} src={governor.avatar} />
+          <ProgressBar style={styles.progressBar} stats={governorStats} />
+          <div style={styles.textSection}>
+            <div style={{...styles.textBox, color: pptColors.primaryRed}}>
+              <div style={{fontSize: 34, height: '67%'}}>{governorStats.notyet || 0}</div>
```

Comment:
Originally, I thought it should be responsive as well, but seems it will not work here.
fixed